cmd/minishift/cmd/root.go

```diff
@@ -239,7 +294,7 @@ func performPostUpdateExecution(markerPath string) error {
 	if markerData.InstallAddon {
 		fmt.Println(fmt.Sprintf("Minishift was upgraded from v%s to v%s. Running post update actions.", markerData.PreviousVersion, version.GetMinishiftVersion()))
 		fmt.Print("--- Updating default add-ons ... ")
-		util.UnpackAddons(constants.MakeMiniPath("add-ons"))
+		util.UnpackAddons(minishiftDirs.Addons)
```

Comment:
this will eventually make the code more readable and avoids having to address our constants.